script/Arcueid/Day 10/10_00_ARC10_7.txt

```diff
@@ -18,7 +18,7 @@ Also you ain't got the skills to take down this kid.
 -- Page 756, Offset 26324.
 -- 「なんでって、そろそろ<調律|ピアノ>の影響が出る頃だからだよ。
 // His gloves are the 'Piano Machine'
-"Well, 'cause the effects of the Piano's 'boutta kick in any minute, that's why.
+"Well, 'cause the effects of the %{i}Piano's%{/i} 'boutta kick in any minute, that's why.
```

Comment:
looks a bit weird to be with the `'s` in there, might just not italics this one and let the capitals indicate it's Special